client/src/Components/Common/Icon/Email.js

```diff
@@ -1,9 +1,16 @@
 import React from "react";
 
 const Email = props => (
-  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" height={props.height} width={props.width} {...props}>
-  <path d="M467 61H45C20.218 61 0 81.196 0 106v300c0 24.72 20.128 45 45 45h422c24.72 0 45-20.128 45-45V106c0-24.72-20.128-45-45-45zm-6.214 30L256.954 294.833 51.359 91h409.427zM30 399.788V112.069l144.479 143.24L30 399.788zM51.213 421l144.57-144.57 50.657 50.222c5.864 5.814 15.327 5.795 21.167-.046L317 277.213 460.787 421H51.213zM482 399.787L338.213 256 482 112.212v287.575z" fill={props.color || "currentColor"}/>
-</svg>
+  <svg
+    enableBackground="new 0 0 479.058 479.058"
+    viewBox="0 0 479.058 479.058"
+    xmlns="http://www.w3.org/2000/svg"
+    height={props.height}
+    width={props.width}
+    {...props}
+  >
+    <path d="m434.146 59.882h-389.234c-24.766 0-44.912 20.146-44.912 44.912v269.47c0 24.766 20.146 44.912 44.912 44.912h389.234c24.766 0 44.912-20.146 44.912-44.912v-269.47c0-24.766-20.146-44.912-44.912-44.912zm0 29.941c2.034 0 3.969.422 5.738 1.159l-200.355 173.649-200.356-173.649c1.769-.736 3.704-1.159 5.738-1.159zm0 299.411h-389.234c-8.26 0-14.971-6.71-14.971-14.971v-251.648l199.778 173.141c2.822 2.441 6.316 3.655 9.81 3.655s6.988-1.213 9.81-3.655l199.778-173.141v251.649c-.001 8.26-6.711 14.97-14.971 14.97z" />
```

Comment:
If you add a ```fill={props.color || "currentColor"}``` into each path tag within the svg we will in the future be able to dynamically change the colour of the svg

Reply:
done :+1: